docs/root/start/sandboxes/locality_load_balancing.rst

```diff
@@ -0,0 +1,141 @@
+******************************************************************
+
+In this scenario, each locality has 1 healthy replica running. In this case, all the requests should be sent to the locality with the highest priority, which is ``local-1``.
+
+.. code:: shell
+
+    # all requests to local-1
+    $ docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+    Hello from backend-local-1!: 100, 100.0%
+    Failed: 0
+
+If locality ``local-1`` becomes unhealthy, the requests should be load balanced among the subsequent priority localities, which are ``local-2`` and ``remote-1``. They both have priority 1. We then send 100 requests to the backend cluster, and check the responders.
+
+.. code:: shell
+
+    # bring down local-1
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_1:8000/unhealthy
+    [backend-local-1] Set to unhealthy
+
+    # local-2 and remote-1 localities split the traffic 50:50
+    $ docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+    Hello from backend-remote-1!: 51, 51.0%
+    Hello from backend-local-2!: 49, 49.0%
+    Failed: 0
+
+Now if ``local-2`` becomes unhealthy also, priority 1 locality is only 50% healthy. Requests will be sent to the ``remote-1`` and ``remote-2``.
+
+.. code:: shell
+
+    # bring down local-2
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-2_1:8000/unhealthy
+
+    # remote-1 locality receive 100% of the traffic
+    $ docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+    Hello from backend-remote-1!: actual weight 69.0%
+    Hello from backend-remote-2!: actual weight 31.0%
+    Failed: 0
+
+    # recover local-1 and local-2 after the demo
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_1:8000/healthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-2_1:8000/healthy
+
+
+Step 3: Scenario with multiple replica in the highest priority locality
+***********************************************************************
+
+Continue from previous step. We first scale up the ``local-1`` locality to 5 replicas.
+
+.. code:: shell
+
+    $ docker-compose up --scale backend-local-1=5 -d
+
+We are going to show the scenario that ``local-1`` is just partially healthy. So let's bring down 4 of the replicas in ``local-1``.
+
+.. code:: shell
+
+    # bring down local-1 replicas
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_2:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_3:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_4:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_5:8000/unhealthy
```

Comment:
lets put this in a separate code block

Reply:
(for clarity - i meant the bit below)

Reply:
I separated bring down block and healthiness check block